### DISCUSSION TITLE
Odd behavior while homing from Z-Offset menu

### DISCUSSION THREAD

**Discussion by dcgrove** [Jyers/Marlin#1014] [Q&A] [answered]
Hello, I recently added quiet fans and the Fang fan shroud to my Ender 3v2. After getting all put back together, I got the be all leveled, then went to check that my z-offset was correct. When I select "Home Z Axis" the printer goes to the center of the bed, the BLtouch extends the probe twice, and then the print head moves to the rear right hand corner (see video below) where the nozzle is completely off the bed. Any idea what I messed up to make this happen? I have formatted the SD card, flashed the firmware to two different firmwares, and it still happens.

[Weird Z-Offset Behaivor](https://youtu.be/8b5Tb3D3GPc)

**Comment by Jyers** [comment, marked as answer]
Well that's not good is it 😅 

Try clearing eeprom by going to the control menu and selecting reset to default and then store settings. Sometimes old eeprom can cause very strange issues.

**Reply by dcgrove** [reply]
Thanks! The issue went away after I flashed the latest firmware.
